proto/spacemesh/v1/debug.proto

```diff
@@ -13,4 +13,7 @@ service DebugService {
     // This includes each account's address, nonce and balance but excludes projection of account state.
     rpc Accounts (google.protobuf.Empty) returns (AccountsResponse);
 
+    // PublicKey returns this node's public key
```

Comment:
Could you expand this comment to explain exactly which public key is returned? go-spacemesh actually uses several public keys. See https://github.com/spacemeshos/go-spacemesh/issues/2269 for more.